### PR TITLE
Profiles.getCurrent()

### PR DESCRIPTION
**Pull request by sandervanhooft**

### CODE REVIEW

**Comment by sandervanhooft** [comment]
@Smitsel Do we want such a method, if so, should it be called `getCurrent` or `getMe`?

File: tests/Mollie/API/Endpoints/ProfileEndpointTest.php [hunk 1]
```diff
@@ -99,6 +99,95 @@ public function testGetProfile()
...
+                        "self": {
+                            "href": "https://api.mollie.com/v2/profiles/pfl_ahe8z8OPut",
+                            "type": "application/hal+json"
+                        },
+                        "chargebacks": {
+                            "href": "https://api.mollie.com/v2/chargebacks?profileId=pfl_ahe8z8OPut",
+                            "type": "application/hal+json"
+                        },
+                        "methods": {
+                            "href": "https://api.mollie.com/v2/methods?profileId=pfl_ahe8z8OPut",
```
Comment (Smitsel) [inline tests/Mollie/API/Endpoints/ProfileEndpointTest.php]: I do not have the `?profileId=` query string parameter in my actual response when i call `/v2/profiles/me`

File: src/Endpoints/ProfileEndpoint.php [hunk 2]
```diff
@@ -50,19 +50,32 @@ public function create(array $data = [], array $filters = [])
...
 
+    /**
+     * Retrieve the current Profile from Mollie.
+     *
+     * @param array $parameters
+     *
+     * @return \Mollie\Api\Resources\BaseResource|\Mollie\Api\Resources\Profile
+     * @throws ApiException
+     */
+    public function getCurrent(array $parameters = [])
```
Comment (Smitsel) [inline src/Endpoints/ProfileEndpoint.php]: I think this name is fine ! :)

File: tests/Mollie/API/Endpoints/ProfileEndpointTest.php (hunk 1 above)
Comment (sandervanhooft) [reply tests/Mollie/API/Endpoints/ProfileEndpointTest.php]: I see, the [docs](https://docs.mollie.com/reference/v2/profiles-api/get-profile-me#example) agree with you!

So I should adhere to the docs, or have we discovered an API issue here? (Don't think so)

Comment (sandervanhooft) [reply tests/Mollie/API/Endpoints/ProfileEndpointTest.php]: I'll remove the link params for now, let me know if things change :)

**Comment by sandervanhooft** [comment]
@Smitsel Fixed

**Pull request merged**